scipy/linalg/tests/test_autosolve.py

```diff
@@ -0,0 +1,285 @@
+# coverage test for scipy autosolve ... about 99%
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.linalg import hilbert
+from autosol import (autosolve, autosolve_nonneg,
+                     autosolve_rising,two_norm)
+# when autosol is online...
+# from scipy.autosol import (autosolve, autosolve_nonneg,
+#                   autosolve_rising, two_norm)
+
+
+# inconsistent arrays .. these crash the run
+# A=np.zeros((3,3))
+# b=np.zeros(5)
+# x=autosolve(A,b)[0]
+# x=autosolve_nonneg(A,b)[0]
+# x=autosolve_rising(A,b)[0]
+
+# 0-D
+print()

```

Comment:
Tests never prints out anything. You can check the other test files for examples for example `linalg/tests/test_basic.py` for inspiration. The results should be asserted and test suite will pickup the success/fail signals automatically.